src/main/java/io/cdap/e2e/pages/actions/CdfPipelineRunAction.java

```diff
@@ -88,4 +88,18 @@ public static String captureRawLogs() {
     SeleniumDriver.getDriver().switchTo().window(parent);
     return logs;
   }
+
+  public static int getBQInCount() {
```

Comment:
Why it's named "BQ"? Is it specific to BQ? IF so, it should not belong to common class